Lib/idlelib/help.py

```diff
@@ -150,6 +157,58 @@ def handle_data(self, data):
             self.text.insert('end', d, (self.tags, self.chartags))
 
 
+class FontSizer:
+    "Support dynamic widget font resizing."
+    def __init__(self, widget, callback=None):
+        """"Add font resizing functionality to widget.
+
+        Args:
+            widget: Tk widget with font attribute to size.
+            callback: Function to call for additional font resizing
+                based on widget's font attribute.
+        """
+        self.widget = widget
+        self.callback = callback
+        self.bind_events()
+
+    def bind_events(self):
+        "Bind events to the widget."
+        shortcut = 'Command' if darwin else 'Control'
+        # Zoom out works with or without shift.
+        self.widget.event_add(
+                '<<zoom-text-out>>',
+                *[f'<{shortcut}-Key-equal>', f'<{shortcut}-Key-plus>'])
```

Comment:
f(*[1,2]) is exactly the same as f(1,2).  Using * in a call is for when one has a name, not a display.
so pass `f'<{shortcut}-Key-equal>', f'<{shortcut}-Key-plus>'`.
I would still like to remove the pseudoevents.

Reply:
I tried back in May and I've been trying for a few days to get the tests to work without the virtual events.  Even though it seems like the text widget has focus, doing a `text.event_generate('<Control-Key-plus>')` just won't activate the binding.  I'm sure I'm just missing something obvious, but I cannot figure out what it is.  :-(